docs/history.rst

```diff
@@ -1,6 +1,36 @@
 Change Log
 ==========
 
+2.2.0
+~~~~~
+* Minimum PROJ version is 6.1.0
+* `pyproj.crs` updates:
+    * Updated CRS repr (issue #264)
+    * Add Datum, CoordinateSystem, CoordinateOperation clases (issue #262)
+    * Added CRS mapping to/from CF 1.8 (pull #244)
+    * Added to_proj4_dict (issue #226)
+    * Provide option to "pretty print" WKT (issue #258)
+    * Better handle CompoundCRS (issue #265)
+    * Add consistency for `is_geographic` and `is_projected` (issue #274)
+    * Added check in CRS.to_epsg for when proj_list is null (issue #257)
+    * Disallow creation of non*CRS (eg pipeline) in CRS class (issue #267)
+    * Added support for ITRF, compound EPSG, and urn projection strings in CRS (pull #289)
+    * Remove unused _CRS.proj_version from pxd file (pull #255)
+* `pyroj.transformer` updates:
+    * Added `always_xy` option to Transformer (issue #225)
+    * Provide direction option in Transformer.transform (issue #266)
+    * Undo deprecation of `from_crs` (issue #275)
+    * Fix false positive errors raised in transformer (issue #249)
+* Fix `pyproj.Proj` initialization from DerivedGeographicCRS (issue #270)
+* Add interface to get the projection/ellps/prime_meridian/units lists (issue #251)
+* Docs/Build/Test fixes:
+    * Set up automated doc builds with travis / doctr (pull #278)
+    * Replace deprecated time.clock() in tests (pull #245)
+    * don't define unused Sphinx html_static_path (pull #248)
```

Comment:
Also here, I think the above two are not worth mentioning 

(it depends of course on what the purpose of putting it here is. But I see this mainly as a communication to users (about relevant changes they want to know about and not just a listing of all changes), and then the sphinx html_static_path is irrelevant to them)

Reply:
You are correct.. But, for changes I didn't make, I prefer to add it as it gives an extra pointer to the person who made the change as a minor way of appreciation.

Reply:
Maybe I should update it to be more like xarray: http://xarray.pydata.org/en/stable/whats-new.html

Reply:
If appreciation is the main concern (and which is an important concern!), I think adding a list of contributors to this release mentioning their names would actually do more (or at least in addition)

Reply:
That is a good idea. Where is that normally put? Should I add it to the what's new or is there a better spot?

Reply:
Found it for geopandas: https://github.com/geopandas/geopandas/releases

I think I like a mix of what is going on in xarray as well as the release credits of geopandas.